observations/forward_operators/obs_def_1d_state_mod.html

```diff
@@ -32,84 +32,83 @@ <H1>MODULE <em class=program>obs_def_1d_state_mod</em></H1>
 <H2>Overview</H2>
 
 <P>
-There is a flexible mechanism built into the DART framework
-for defining, at compile time, the list of observation types
-to be supported by the executables.  This can be changed at
-any time by adding or removing items from a namelist and 
-rebuilding the programs.  The
-special obs_def module being described here is used by the program 
-<em class="unix">preprocess</em>
-to insert appropriate code sections into a 
-<a href="DEFAULT_obs_def_mod.html">DEFAULT_obs_def module</a>
-and a 
-<a href="../../assimilation_code/modules/observations/DEFAULT_obs_kind_mod.html">DEFAULT_obs_kind module</a>
+The list of observation types to be supported by the DART executables is defined
+at compile time. The observations DART supports can be changed at  any time by adding or
+removing items from the preprocess namelist and rerunning <em>quickbuild.csh</em>.
```

Comment:
I don't think there is any documentation in dart that describes building without using quickbuild.csh

````
If preprocess doesn't exist:
csh mkmf_preprocess
make

./preprocess
csh mkmf_whatever
make
````

I don't think this is the place for that documentation, because what we want people to take away from reading obs_def_1d_state_mod.html is how an obs def files works and what should go in there.  

1m34s to run quickbuild.csh for wrf on my laptop.

Reply:
agree.